docs/running-headscale-linux.md

```diff
@@ -30,6 +30,8 @@ mkdir -p /etc/headscale
 
 # Directory for Database, and other variable data (like certificates)
 mkdir -p /var/lib/headscale
+# or if you create a headscale user:
+useradd -md /var/lib/headscale/ -rUs /usr/bin/nologin headscale
```

Comment:
For documentation, I believe we should use the long form so users understand what they are inputing,

```suggestion
useradd \
  --create-home \
  --home-dir /var/lib/headscale/ \
  --system \
  --user-group\
  --shell /usr/bin/nologin \
  headscale
```

Reply:
fair enough.